File: ARCADE/Intro/Level 9_5: bishopAndPawn.cs
    using System;
     
    public class Test
    {
    	public static bool bishopAndPawn(string bishop, string pawn){
            return (Math.Abs(bishop[0] - pawn[0]) == Math.Abs(bishop[1] - pawn[1]));
        }
    	public static void Main()
    	{
    		string bishop = "a1", pawn = "c3";
    		Console.WriteLine(bishopAndPawn(bishop, pawn));
    	}
    }

File: ARCADE/Intro/Level 9_5: bishopAndPawn.pas
program bishopAndPawn;
 
uses Math;
 
function bishopAndPawn(bishop: string; pawn: string): boolean;
begin
    if(Abs(ord(bishop[1]) - ord(pawn[1])) = Abs(ord(bishop[2]) - ord(pawn[2]))) then bishopAndPawn := true
    else bishopAndPawn := false;
end;
 
var bishop, pawn: string;
begin
	bishop := 'a1'; pawn := 'c3';
	writeln(bishopAndPawn(bishop, pawn));
 
	bishop := 'h1'; pawn := 'h3';
	writeln(bishopAndPawn(bishop, pawn));
end.
